_posts/2022-12-01-predict-proba.md

```diff
@@ -0,0 +1,67 @@
+---
+#### Blog Post Template ####
+
+#### Post Information ####
+# title: "What we should and should not expect from `predict_proba`"
+title: "What to expect from `predict_proba`"
+date: December 1, 2022
+
+#### Post Category and Tags ####
+# Format in titlecase without dashes (Ex. "Open Source" instead of "open-source")
+categories:
+  - Updates
+tags:
+  - Machine Learning
+
+#### Featured Image ####
+featured-image: sorting.png
+
+#### Author Info ####
+# Can accomodate multiple authors
+# Add SQUARE Author Image to /assets/images/author_images/ folder
+postauthors:
+  - name: Alexandre Perez-Lebel
+    website: https://perez-lebel.com
+    email: alexandre.perez@inria.fr
+    image: alexandre_perez.jpeg
+usemathjax: true
+---
+<div>
+  <img src="/assets/images/posts_images/{{ page.featured-image }}" alt="" title="Image by storyset on Freepik">
+  {% include postauthor.html %}
+</div>
+
+In classification, many situations call for estimated probabilities beyond class labels. For example in decision making, cost sensitive learning or causal inference.
+These probability estimates are typically accessible from the `predict_proba` method of scikit-learn's classifiers.
+
+However, the quality of the estimated probabilities must be validated to provide trustworthiness, ensure fairness and robustness to operating conditions.
+To be reliable, the estimated probabilities must be close to the true underlying posterior probabilities of the classes `P(Y=1|X)`.
```

Comment:
As you know, there are several points of view on what such a probability may mean (controlled average error rate versus controlled individual probabilities). Maybe it would be good by first explaining what these two mean.